concordia/static/js/action-app/index.js

```diff
@@ -792,6 +805,28 @@ export class ActionApp {
         return sortBy(assetList, keyFromAsset);
     }
 
+    getCampaignOrTopic() {
```

Comment:
I'm wondering about the name for this – it read at first like an accessor but it's really something which gets us the type of the currently selected `<option>`. Maybe something like `getSelectedOptionType()`?

Reply:
Done